packages/quantic/cypress/reporters/detailed-reporter.js

```diff
@@ -154,6 +158,16 @@ class DetailedReporter {
     );
   }
 
+  /**
+   * Handles the `action` message.
+   * @param {Message} message The message instance.
+   */
+  handleAction(message) {
+    this._logBuffer.push(
+      this.scopeIndent() + chalk.blue('-> ')+ chalk.dim(message.content)
+    )
+  }
```

Comment:
Using 2 characters to display the arrow will shift the message to the right and make it misaligned with the expectation messages. I'd suggest to use a single character for the symbol (e.g., `→`, or `>` for Windows).

```suggestion
  handleAction(message) {
    const symbol = this._isWindows ? '>' : '→';
    this._logBuffer.push(
      `${this.scopeIndent()}${chalk.blue(symbol} ${chalk.dim(message.content)}`
    )
  }
```